src/pages/en/guides/typescript.md

```diff
@@ -78,11 +78,38 @@ export interface Props {
   name: string;
   greeting?: string;
 }
-const { greeting = 'Hello', name } = Astro.props as Props
+const { greeting = 'Hello', name } = Astro.props as Props;
 ---
 <h2>{greeting}, {name}!</h2>
 ```
 
+### Built-in attribute types
+
+Astro provides JSX type definitions to check your markup is using valid HTML attributes. You can use these types to help build component props. For example, if you were building a `<Link>` component, you could do the following to mirror the default HTML attributes in your component’s prop types.
+
+```astro
+---
+export type Props = astroHTML.JSX.HTMLAttributes<HTMLAnchorElement>;
```

Comment:
This is using the old `astro-jsx.d.ts` format, the file has significantly changed about a week ago and now has element specific attributes interfaces and doesn't require any generics anymore. For a Link component, you'd now do:

```suggestion
export type Props = astroHTML.JSX.AnchorHTMLAttributes;
```

Reply:
Ah right! I was testing it out in docs and we’re still on beta.64. Thanks for the review.